client/webserver/locales/en-us.go

```diff
@@ -262,4 +262,10 @@ var EnUS = map[string]string{
 	"No Recent Activity":          "No Recent Activity",
 	"Recent asset_name Activity":  "Recent <span data-asset-name=1></span> Activity",
 	"Other Actions":               "Other Actions",
+	"disable_wallet":              "Disable Wallet",
+	"enable_wallet":               "Enable Wallet",
+	"change_wallet_status":        "Change Wallet Status",
+	"disable_wallet_warning":      "Note: This wallet will not be connected to when you start the DEX client software and cannot be used until it is enabled. Ensure you do not have active trades.",
```

Comment:
Backend checks `assetHasActiveOrders` anyway, so that last sentence seems unnecessary.  Plus we could also consider `app().haveAssetOrders(assetID)` on the frontend.